docs/docs/features/key-assignment.md

```diff
@@ -80,3 +80,13 @@ To change your key, simply repeat all of the steps listed above. Take note that
 ## Removing a key
 
 To remove a key, simply switch it back to the consensus key you have assigned on the provider chain by following steps in the `Adding a key` section and using your provider consensus key.
+
+## Querying proposed consumer chains
+
+To query the pending consumer addition proposals, you can use the following command on the provider:
```

Comment:
> the pending consumer addition proposals

I think there is a difference between "pending consumer addition proposals" and consumer addition proposals that are in the voting period. Which one is this?